server/game/cards/09.1-FtE/CommandRespect.js

```diff
@@ -0,0 +1,31 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes, Players } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class CommandRespect extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'takes honors from opponent when they play an event',
+            condition: context => {
+                return context.player.hand.size() < context.player.opponent.hand.size() &&
+                context.game.isDuringConflict();
+            },
+            max: AbilityDsl.limit.perConflict(1),
+            gameAction: AbilityDsl.actions.playerLastingEffect(() => {
+                return {
+                    targetController: Players.Opponent,
+                    effect: AbilityDsl.effects.additionalCost((context) => {
+                        if(context.source.type === CardTypes.Event) {
+                            return [AbilityDsl.costs.giveHonorToOpponent(1)];
+                        }
+                        return [];
+                    })
+                };
```

Comment:
Needs an effect (otherwise you just get 'player1 plays Command Respect to apply a lasting effect' in chat)